lib/helpers/errorHandler.js

```diff
@@ -20,14 +20,20 @@ export default [
             if (command.name !== 'element' && command.name !== 'elements') {
                 continue
             }
-            if (command.name === 'element' && (!command.result[0].value || command.result[0].value.ELEMENT !== failingCommand.args[0])) {
+
+            if (command.name === 'element' && (!command.result || !command.result.value || command.result.value.ELEMENT !== failingCommand.args[0])) {
                 continue
             }
 
-            for (let result of command.result.value) {
-                if (result.ELEMENT === failingCommand.args[0]) {
-                    commandToRepeat = this.commandList[i - 1]
-                    break
+            if (command.result) {
+                // Ensure an array to simply the below logic
+                const results = Array.isArray(command.result.value) ? command.result.value : command.result.value !== null ? [command.result.value] : []
```

Comment:
Thanks for the feedback!

As far as I can tell, the logic here will still accept a match from an 'element' command. In the case of an 'element' command with a result, the ```results``` variable will be an array with just that one result in it. Basically, the array part just means I don't need an ```if``` for the 'element' case. The amount of simplification that accomplishes is very, very little though... I would be fine just using an ```if``` like you did too.